src/weapon.py

```diff
@@ -4,19 +4,61 @@
 import random
 
 class Weapon(Item):
-    def __init__(self, name, desc, value, damage):
+    def __init__(self, damage, name):
         """@ReturnType Weapon"""
-        self.___damage = damage
-        self.___name = name
+        self.__damage = damage
         """@AttributeType int"""
-        super(Weapon, self).__init__(name, desc, value)
+        self.__name = name
+        """@AttributeType String"""
+        self.__value = damage
+        """@AttributeType int"""
+        self.__desc = "A weapon to be used in combat. Equip to increase your damage."
+        """@AttributeTupe String"""
+        super(Weapon, self).__init__(name, self.__desc, self.__value)
```

Comment:
The Weapon constructor shouldn't be setting name, value or description, because those are handled by the Item constructor. Remove those parameters from Weapon's __init__ and pass them to the Super call.

Reply:
I fixed it. Thank you! The newest commit has the updates